apps/server/src/modules/tool/context-external-tool/service/tool-version-service.ts

```diff
@@ -25,13 +25,29 @@ export class ToolVersionService {
 	): Promise<ToolConfigurationStatus> {
 		// TODO N21-1337 remove if statement, when feature flag is removed
 		if (this.toolFeatures.toolStatusWithoutVersions) {
+			const configurationStatus: ToolConfigurationStatus = new ToolConfigurationStatus({
+				latest: true,
```

Comment:
Do we really need to set isLatest or can you use logic in the domain object?
For example isnt it the latest if isOutdatedOnScopeContext and isOutdatedOnScopeSchool is false?

Reply:
I use the the logic way now, thanks :)